lib/cron_kubernetes/cron_job.rb

```diff
@@ -44,6 +37,19 @@ def namespace
       "default"
     end
 
+    def build_cron_job_spec
+      default_spec = {
+          "schedule" => schedule,
+          "concurrencyPolicy" => "Forbid",
```

Comment:
Hard-coding this into the spec seems like a bad ideas. Wouldn't it be possible that folks using this would want to change this value?